hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -125,10 +129,32 @@ instance FromCBOR Tx where
         (pure . fromLedgerTx)
 
 instance ToJSON Tx where
-  toJSON = toJSON . toLedgerTx
+  toJSON tx =
+    let TextEnvelopeType envelopeType = textEnvelopeType (proxyToAsType (Proxy @Tx))
+     in object
+          [ "cborHex" .= (Aeson.String $ decodeUtf8 $ Base16.encode $ serialiseToCBOR tx)
+          , "txId" .= (txId tx)
+          , "type" .= envelopeType
+          ]
```

Comment:
Could: piggy back on the cardano-api `TextEnvelope` encoding

We did this on the `/commit` [endpoint response](https://github.com/input-output-hk/hydra/blob/66c431f94985274626b82d949a66e820db3b432b/hydra-node/src/Hydra/API/HTTPServer.hs#L58) (which could be removed in favor of just using this instance then).

The same holds for the `parseJSON` which could deserialize the envelope + the txId decoding + checking